respy/simulate.py

```diff
@@ -152,11 +152,14 @@ def _simulate_data(state_space, base_draws_sim, base_draws_wage, optim_paras, op
     for lag in reversed(range(1, n_lagged_choices + 1)):
         container += (_get_random_lagged_choices(states_df, optim_paras, options, lag),)
 
+    for observable in optim_paras["observables"].keys():
```

Comment:
This is going to be a little bit weird, but I think we have to move this call even before sampling experiences. The reason is that in KW2000 the observable (race) influences the distribution of initial schooling levels.

At the same time we should add observables between lagged choices and types in the state space, because our current layout depends on it and we should not rework it now.